### ISSUE TITLE
Typo in bulkRename.py code

### ISSUE THREAD

**Issue by primowalker** [thewhitetulip/SamplePythonScripts#1]
There is a typo on line 19 of bulkRename.py.  What you have there is:

os.rename(i,template_name+ str(j)+template_extension')

The issue is that the single quote is missing at the beginning inside the parentheses, so it should read:

os.rename('i,template_name+ str(j)+template_extension')

**Comment by thewhitetulip** [comment]
Nope, there should not be a single quote, `os.rename` takes two arguments the old name and the new name, if we surround it by single quotes, it becomes only one argument.
